src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.test.tsx

```diff
@@ -58,6 +58,9 @@ const mockState = {
       human: {
         status: Status.OPEN
       }
+    },
+    bookmarks: {
+      previouslyViewed: {}
```

Comment:
I believe this addition to the mock state is a remnant from Ridwan's PR? There is no longer any dependency of TranscriptFilters on this piece of the redux state, right?

Reply:
Yes! I've removed it.